nano/secure/common.hpp

```diff
@@ -264,8 +264,15 @@ class vote final
 	boost::transform_iterator<nano::iterate_vote_blocks_as_hash, nano::vote_blocks_vec_iter> begin () const;
 	boost::transform_iterator<nano::iterate_vote_blocks_as_hash, nano::vote_blocks_vec_iter> end () const;
 	std::string to_json () const;
+	uint64_t timestamp () const;
+	static uint64_t constexpr timestamp_max = { 0xffff'ffff'ffff'ffffULL };
```

Comment:
Would it be maybe any better to use the numeric limits? 

Also, on a different note, is `timestamp_m` (the suffix `_m`) for member? I don't remember seeing it a lot.

Reply:
This number will be different with the actual vote duration changes.